### PR TITLE
Add ability to change a DVS that has already been created

### PR DESCRIPTION
**Pull request by thadc23**

### CODE REVIEW

File: govc/dvs/change.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+			}
+
+			cmd.configSpec.ConfigVersion = s.Config.GetDVSConfigInfo().ConfigVersion
+			task, err := dvs.Reconfigure(ctx, cmd.ConfigSpec)
+			if err != nil {
+				return err
+			}
+
+			logger := cmd.ProgressLogger(fmt.Sprintf("updating %s in folder %s... ", name, folder.InventoryPath))
+			defer logger.Wait()
```
Comment (dougm) [inline govc/dvs/change.go]: Note that if multiple arguments are provided, the ProgressLogger doesn't display properly.. would need to move these calls into a separate func for defer to work, or call `logger.Wait()` explicitly.  We likely have this bug in a few other commands.  Another reason to keep it simple and just support a single object :)

File: govc/dvs/change.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	if err != nil {
+		return err
+	}
+
+	folder, err := cmd.FolderOrDefault("network")
+	if err != nil {
+		return err
+	}
+
+	networks, err := finder.NetworkList(ctx, folder.InventoryPath+"/"+name)
```
[the remark this answers is not in the document]
Comment (thadc23) [reply govc/dvs/change.go]: I totally missed `finder.Network()`. Using it now.

File: govc/dvs/change.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+func (cmd *change) Usage() string {
+	return "DVS"
+}
+
+func (cmd *change) Description() string {
+	return `Change DVS (DistributedVirtualSwitch) in datacenter.
+
+Examples:
+  govc dvs.change
```
[the remark this answers is not in the document]
Comment (thadc23) [reply govc/dvs/change.go]: done

File: govc/dvs/change.go [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+		return err
+	}
+
+	networks, err := finder.NetworkList(ctx, folder.InventoryPath+"/"+name)
+	if err != nil {
+		return err
+	}
+
+	for _, net := range networks {
+		if dvs, ok := net.(*object.DistributedVirtualSwitch); ok {
```
[the remark this answers is not in the document]
Comment (thadc23) [reply govc/dvs/change.go]: fixed now that I am not looping to find the network

File: govc/dvs/change.go (hunk 1 above)
Comment (thadc23) [reply govc/dvs/change.go]: Fixed.  I only ever meant to update a single DVS at a time

**Comment by thadc23** [comment]
@dougm Hope this looks better.  Let me know if there is anything else.

**Pull request merged**